docker/bin/push2dockerhub.sh

```diff
@@ -5,10 +5,6 @@ set -ex
 BIN_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source $BIN_DIR/set_git_env_vars.sh
 
-DOCKER_USERNAME="${DOCKER_USERNAME:-mozjenkins}"
-
-docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
```

Comment:
Assuming these are now set in the cluster?

Reply:
the credentials are stored on the runner instances and interactive login using a password from an env var is neither needed nor desired.